src/operations-tracker-body/operations-tracker-body.tsx

```diff
@@ -229,11 +229,17 @@ const getTabHeaderName = (
   state: ICountReducerState,
   data: IDataView
 ) => {
+  const lengthOfObject = data.affectedQueriesOperations
```

Comment:
useMemo

Reply:
but its not heavy computation, do we need to memo?

Reply:
It'll run on every render. And grid keeps rendering all the time. Here you're getting the keys object all the time.